### ISSUE TITLE
CLI : new option to write a file with all COPY dependencies

### ISSUE THREAD

**Issue by smedilol** [TypeCobolTeam/TypeCobol#454]
# The need
In the context of our private RDZ plugin which allows us to write TypeCobol code and generate it:
Our parser need all COPY files to be accessible so all symbols can be resolved.
This means we need to retrieve all COPY files from our SCM. This COPY need to be located into the SYSLIB folder (located in the RDZ workspace in `RemoteSystemsTempFiles\FttRemoteTempFiles\nitcpip.e-i.net\ED\ED.D$RDZ.SYSLIB`).

In order to that, we need an option in TypeCobol CLI to write a new file with all COPY dependencies **which are missing** (see format below).
The new file will then be parsed by our private RDZ plugin to download all COPYs into the SYSLIB folder 

In other words, the new file will only contains COPY related to errors "Failed to load the text document referenced by a COPY directive".

The new option will be `--HaltOnMissingCopy".

# COPY file format
This will be a simple text file.
One COPY per line.

# Implementation notes
When this new option is specified, the parsing must halt just before CodeElement phase if at least one copy is missing.
Only execute steps `UpdateTokensLines` to `RefreshProcessedTokensDocumentSnapshot`
Steps `RefreshCodeElementsDocumentSnapshot` and `RefreshProgramClassDocumentSnapshot` should not be necessary.

**Comment by collarbe** [comment]
To use the new option add -hc when calling TypeCobol.CLI and pass the file name to create. `-hc "copies.txt"`
The file will be created in : `C:\Users\<USERAME>\AppData\Local\Temp\tcbl`

**Comment by smedilol** [comment]
One thing I forgot:
If for one parsing, the file is generated with missing copies and the next parsing no copies are missing, then the missing copy files still contains the list of missing copies from the 1st parsing.

We need to the delete the file if no copies are missing.
